src/vtkF3DAlembicImporter.cxx

```diff
@@ -0,0 +1,351 @@
+// #include <Alembic/AbcCoreHDF5/All.h> Not sure if we want to support legacy HDF5 stuff
+#include <Alembic/AbcCoreOgawa/All.h>
+#include <Alembic/Util/All.h>
+#include <Alembic/Abc/All.h>
+#include <Alembic/AbcCoreAbstract/All.h>
+#include <Alembic/AbcCoreFactory/All.h>
+
+vtkStandardNewMacro(vtkF3DAlembicImporter);
+
+
+class vtkF3DAlembicImporterInternalEx
+{
+	typedef std::map<Alembic::AbcGeom::PlainOldDataType,std::string> PODStringMap;
+	typedef std::map<Alembic::AbcGeom::PropertyType,std::string> PropertyTypeStringMap;
+public:
+	vtkF3DAlembicImporterInternalEx(vtkF3DAlembicImporter* parent) {
+		this->Parent = parent;
+	}
+	void CreatePODStringMap(PODStringMap& podStringMap)
+	{
+		podStringMap[Alembic::AbcGeom::kBooleanPOD] = "kBooleanPOD";
+		podStringMap[Alembic::AbcGeom::kUint8POD] = "kUint8POD";
+		podStringMap[Alembic::AbcGeom::kInt8POD] = "kInt8POD";
+		podStringMap[Alembic::AbcGeom::kUint16POD] = "kUint16POD";
+		podStringMap[Alembic::AbcGeom::kInt16POD] = "kInt16POD";
+		podStringMap[Alembic::AbcGeom::kUint32POD] = "kUint32POD";
+		podStringMap[Alembic::AbcGeom::kInt32POD] = "kInt32POD";
+		podStringMap[Alembic::AbcGeom::kUint64POD] = "kUint64POD";
+		podStringMap[Alembic::AbcGeom::kInt64POD] = "kInt64POD";
+		podStringMap[Alembic::AbcGeom::kFloat16POD] = "kFloat16POD";
+		podStringMap[Alembic::AbcGeom::kFloat32POD] = "kFloat32POD";
+		podStringMap[Alembic::AbcGeom::kFloat64POD] = "kFloat64POD";
+		podStringMap[Alembic::AbcGeom::kStringPOD] = "kStringPOD";
+		podStringMap[Alembic::AbcGeom::kWstringPOD] = "kWstringPOD";
+		podStringMap[Alembic::AbcGeom::kNumPlainOldDataTypes] = "kNumPlainOldDataTypes";
+		podStringMap[Alembic::AbcGeom::kUnknownPOD] = "kUnknownPOD";
+	}
+	void CreatePropertyTypeStringMap(PropertyTypeStringMap& propertyTypeStringMap)
+	{
+		propertyTypeStringMap[Alembic::AbcGeom::kCompoundProperty] = "kCompoundProperty";
+		propertyTypeStringMap[Alembic::AbcGeom::kScalarProperty] = "kScalarProperty";
+		propertyTypeStringMap[Alembic::AbcGeom::kArrayProperty] = "kArrayProperty";
+	}
+	void ProcessIPolyMesh(vtkRenderer* renderer, const Alembic::AbcGeom::IPolyMesh& pmesh)
+	{
+		vtkSmartPointer<vtkPoints> points = vtkSmartPointer<vtkPoints>::New();
+		vtkSmartPointer<vtkCellArray> polys = vtkSmartPointer<vtkCellArray>::New();
+		vtkSmartPointer<vtkPolyData> polydata = vtkSmartPointer<vtkPolyData>::New();
+
+		Alembic::AbcGeom::IPolyMeshSchema::Sample samp;
+		if ( pmesh.getSchema().getNumSamples() > 0 )
+		{
+			pmesh.getSchema().get( samp );
+
+			Alembic::AbcGeom::P3fArraySamplePtr P = samp.getPositions();
+			Alembic::AbcGeom::Int32ArraySamplePtr indices = samp.getFaceIndices();
+			Alembic::AbcGeom::Int32ArraySamplePtr counts = samp.getFaceCounts();
+			Alembic::AbcGeom::V3fArraySamplePtr v = samp.getVelocities();
+
+			size_t P_size = P->size();
+			size_t indices_size = indices->size();
+			size_t counts_size = counts->size();
+
+			std::cout << "P size = " << P_size << std::endl;
```

Comment:
You will need to remove all these `std::cout`

Reply:
I have removed all the debugging print out.